src/midiEditorCommands.cpp

```diff
@@ -468,19 +554,15 @@ bool compareNotesByPitch(const MidiNote& note1, const MidiNote& note2) {
 // This updates curNoteInChord.
 MidiNote findNoteInChord(MediaItem_Take* take, int direction) {
 	auto chord = findChord(take, 0);
-	if (chord.first == -1)
+	if (chord.first == -1) {
 		return {-1};
+	}
+	MidiNoteIterator begin(take, MidiNote::GetParams{true, true, true, true, true});
```

Comment:
1. I still think the readability here is a concern. Please comment the param names; e.g. `/* position */ true`.
2. Will the compiler deduce the type if you remove the `MidiNote::GetParams` here, so it's just: `MidiNoteIterator begin(take, {...})`? It might not, but if it does, let's do that.